force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.js

```diff
@@ -98,6 +137,18 @@ export default class NewServiceSchedule extends LightningElement {
             });
     }
 
+    handleLoad() {
+        this.template.querySelectorAll("lightning-input-field").forEach(field => {
+            if (field.fieldName === this.dateFields.start.apiName) {
+                this.setFirstSessionStartTimeAndEndTime(field.value);
+            }
+        });
+
+        this.picklistFields.frequency.value = !this.picklistFields.frequency.value
```

Comment:
Is this still necessary now that you are respecting defaultValues in the picklist component?

Reply:
Yes unfortunately it is necessary else I get an error message saying unable to de-reference a null object in the console since the value is being passed as null